doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,174 @@
+**All Terms utilized throughout this chapter are intended to align with CVC definitions, and their use through CVC documentation, guidelines, and standards.**
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+**Chapter Purpose**<br>
+This chapter includes process flow, logistics, and requirements which must be satisfied to ensure Kubernetes infrastructure meets the design, feature, and capability expectations of the Reference Model (RM) and RA2. Upstream projects will define features/capabilities, test scenarios, and test cases which will be used to test for infrastructure verification purposes. Existing processes, communication mediums, and related technologies will be utilized where feasible.  Ultimately, test results of infrastructures will reduce the amount of time and cost it takes each operator to on-board and maintain vendor provided Kubernetes infrastructure and CNFs.
+
+**Objective**<br>
+Perform Kubernetes infrastructure and CNF Verification and Validations using CNTT RA2 to test code for compliance. Upstream projects will define features/capabilities, test scenarios, and test cases.
+
+**Test Methodology**
+- Verification test to make sure that the Kubernetes services have been deployed and configured correctly
+- Manifest Verifications (Termed Compliance by CVC) will ensure the infrastructure is compliant, and delivered for testing, with hardware and software profile specifications defined by the RM and RA2
+- Candidate CNF Validation will ensure interoperablity of CNF behaviour on the Kubernetes infrastructure.  Testing ensures CNFs can be spun up, modified, or removed, on the target infrastructure (aka Interoperability).  
+
+<a name="1.1.1"></a>
+### 1.1.1 Terminology
+
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+
+<a name="1.2"></a>
+## 1.2 Scope
+
+This document covers the realisation aspects of conformance of both Kubernetes infrastructure and CNFs. The document will cover the following topics:
+
+- Identify in details the Requirement of Test Cases (and mapping them to requirements from The Reference Model and the Kubernetes Based Reference Architecture).
+- Analysis of existing community projects.
+- Propose an E2E Framework for conformance of Kubernetes infrastructures and CNFs.
+- Playbook of instructions, user manuals, steps of how to perform verification and conformance for both infrastructure and CNFs using the proposed E2E Framework.
+- Gap analysis to identify where the Gaps are in the industry (tooling, test cases, process, etc).
+- Identify development efforts needed to address any gaps identified.
+
+**Not in Scope**
+- Functional testing / validation of the application provided by the CNF is outside the scope of this work.
+- ONAP is not used in the process flow for infrastructure verifications, or validations.
+- Upgrades to CNFs, and the respective processes of verifying upgrade procedures and validating (testing) the success and compatibility of upgrades is not in scope.
```

Comment:
Not necessarily suggesting it is out of scope, but performance testing and performance benchmarks are not mentioned either in scope or out of scope.

Reply:
Agree, was leaving this out until we have more discussion on #1495